Code/GraphMol/GenericGroups/GenericGroups.h

```diff
@@ -238,42 +501,167 @@ RDKIT_GENERICGROUPS_EXPORT bool AlkoxyacyclicAtomMatcher(
+        {"AYH", Matchers::AlkynylHAtomMatcher},
         {"Carbocyclic", Matchers::CarbocyclicAtomMatcher},
         {"CBC", Matchers::CarbocyclicAtomMatcher},
+        {"CarbocyclicH", Matchers::CarbocyclicHAtomMatcher},
+        {"CBH", Matchers::CarbocyclicHAtomMatcher},
         {"Carbocycloalkyl", Matchers::CarbocycloalkylAtomMatcher},
         {"CAL", Matchers::CarbocycloalkylAtomMatcher},
+        {"CarbocycloalkylH", Matchers::CarbocycloalkylHAtomMatcher},
+        {"CAH", Matchers::CarbocycloalkylHAtomMatcher},
         {"Carbocycloalkenyl", Matchers::CarbocycloalkenylAtomMatcher},
         {"CEL", Matchers::CarbocycloalkenylAtomMatcher},
+        {"CarbocycloalkenylH", Matchers::CarbocycloalkenylHAtomMatcher},
+        {"CEH", Matchers::CarbocycloalkenylHAtomMatcher},
         {"Carboaryl", Matchers::CarboarylAtomMatcher},
         {"ARY", Matchers::CarboarylAtomMatcher},
+        {"CarboarylH", Matchers::CarboarylHAtomMatcher},
+        {"ARH", Matchers::CarboarylHAtomMatcher},
         {"Cyclic", Matchers::CyclicAtomMatcher},
         {"CYC", Matchers::CyclicAtomMatcher},
+        {"CyclicH", Matchers::CyclicHAtomMatcher},
+        {"CYH", Matchers::CyclicHAtomMatcher},
         {"Acyclic", Matchers::AcyclicAtomMatcher},
         {"ACY", Matchers::AcyclicAtomMatcher},
+        {"AcyclicH", Matchers::AcyclicHAtomMatcher},
+        {"ACH", Matchers::AcyclicHAtomMatcher},
         {"Carboacyclic", Matchers::CarboacyclicAtomMatcher},
         {"ABC", Matchers::CarboacyclicAtomMatcher},
+        {"CarboacyclicH", Matchers::CarboacyclicHAtomMatcher},
+        {"ABH", Matchers::CarboacyclicHAtomMatcher},
         {"Heteroacyclic", Matchers::HeteroacyclicAtomMatcher},
         {"AHC", Matchers::HeteroacyclicAtomMatcher},
+        {"HeteroacyclicH", Matchers::HeteroacyclicHAtomMatcher},
+        {"AHH", Matchers::HeteroacyclicHAtomMatcher},
         {"Alkoxy", Matchers::AlkoxyacyclicAtomMatcher},
         {"AOX", Matchers::AlkoxyacyclicAtomMatcher},
+        {"AlkoxyH", Matchers::AlkoxyacyclicHAtomMatcher},
+        {"AOH", Matchers::AlkoxyacyclicHAtomMatcher},
         {"Heterocyclic", Matchers::HeterocyclicAtomMatcher},
         {"CHC", Matchers::HeterocyclicAtomMatcher},
+        {"HeterocyclicH", Matchers::HeterocyclicHAtomMatcher},
+        {"CHH", Matchers::HeterocyclicHAtomMatcher},
         {"Heteroaryl", Matchers::HeteroarylAtomMatcher},
         {"HAR", Matchers::HeteroarylAtomMatcher},
+        {"HeteroarylH", Matchers::HeteroarylHAtomMatcher},
+        {"HAH", Matchers::HeteroarylHAtomMatcher},
         {"NoCarbonRing", Matchers::NoCarbonRingAtomMatcher},
         {"CXX", Matchers::NoCarbonRingAtomMatcher},
+        {"NoCarbonRingH", Matchers::NoCarbonRingHAtomMatcher},
+        {"CXH", Matchers::NoCarbonRingHAtomMatcher},
+        {"D", Matchers::DAtomMatcher},
+        {"T", Matchers::TAtomMatcher},
+        {"H+", Matchers::HplusAtomMatcher},
+        {"Pol", Matchers::PolAtomMatcher},
+        {"R", Matchers::RAtomMatcher},
```

Comment:
I think these are all "normal" atoms and should be handled as such. They aren't really queries.
What's the motivation for adding them here?

Reply:
I don't think we use these inside sgroups, so we can probably remove them from that PR.

Reply:
These are on the list of expanded groups inside Ketcher editor